### DISCUSSION TITLE
Reverse order per `term`

### DISCUSSION THREAD

**Discussion by joelouthan** [adityatelange/hugo-PaperMod#368] [General]
My site repo: https://github.com/joelouthan/theologic.us

Asked here how to reverse order per taxonomy:
https://discourse.gohugo.io/t/is-it-possible-to-pages-bydate-reverse-for-certain-specific-taxonomies/32390

The last theme I used (cupper) I was able to use the Hugo example to reverse the list order on the pages pretty easily.

Within PaperMod, I cannot figure that out very easily.

In the Hugo support, the replier suggested to do so per `term` (I.e. family-devotions-mark) rather than `taxonmomy` (I.e. series).

Can you assist please? Thank you.

**Comment by adityatelange** [comment]
> The last theme I used (cupper) I was able to use the Hugo example to reverse the list order on the pages pretty easily.

Can you tell us how you have done it with cupper ?

**Reply by joelouthan** [reply]
layouts/_default/list.html
```html
{{ define "main" }}
<main id="main">
  <h1>{{ .Title }}</h1>
  {{ if site.Params.search }}
  <input
    id="search"
    type="text"
    placeholder="{{ T "search_placeholder" }}"
    aria-label="{{ T "search_aria_label" }}"
  />
  {{ end }}
  <ul class="patterns-list" id="list">
    {{ range .Pages.ByPublishDate.Reverse }}
    <li>
      <h2>
        <a href="{{ .RelPermalink }}">
          <svg
            class="bookmark"
            aria-hidden="true"
            viewBox="0 0 40 50"
            focusable="false"
          >
            <use xlink:href="#bookmark"></use>
          </svg>
          {{ .Title }}
        </a>
      </h2>
    </li>
    {{ end }}
  </ul>
</main>
{{ end }}
```

**Reply by joelouthan** [reply]
As you can see, it would have been for all lists.

**Reply by adityatelange** [reply]
Hugo offers 2 separate templates for `term` and `taxonomy`.
We already have `terms` and `list` is used for `taxonomy` instead of `layouts/_default/taxonomy.html`. 

You can create `layouts/_default/taxonomy.html` in your site root and use the `list.html` template's content in `layouts/_default/taxonomy.html`. 

I think you can use `$paginator.Pages.ByPublishDate.Reverse` [here](https://github.com/adityatelange/hugo-PaperMod/blob/fb0393fa125f04e95d54d8ed32ed817e861876da/layouts/_default/list.html#L32)  to get your taxonomies in reverse order.

**Reply by joelouthan** [reply]
@adityatelange Thank you so much.

I have tried setting up like you say and I don't get the effect that I want.

**In an attempt to troubleshoot**: when I edit the `layouts/_default/list.html` and add `$paginator.Pages.ByPublishDate.Reverse`, restart Hugo, and the list of posts are still newest first.

**Update**: On a hunch, I went ahead and not use the `.Reverse` I wanted the posts listed oldest first. Hence, I need to do `$paginator.Pages.ByPublishDate` and not use Reverse. 

That worked!

So I restored `list.html` back to original. Then I make the changes I want within `layouts/_default/series.html` and that worked! So I know how to do that in taxonomies.

Is it possible to do it per specific `term` (example: bible-study-romans) rather than the taxonomy (example: series or tags or categories)?
